### DISCUSSION TITLE
[JEST 28] SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by GustavoSchuler** [uuidjs/uuid#625] [Q&A] [answered]
Hey! I'm facing this error since I've updated to JEST v28.

 _/Users/schuler/dev/madmax-front/node_modules/uuid/dist/esm-browser/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){export { default as v1 } from './v1.js';
                                                                                      ^^^^^^
   SyntaxError: Unexpected token 'export'_

My dependencies are:

```
{
  "next": "^12.1.5",
  "react": "^17.0.2",
  "react-dom": "^17.0.2",
  "uuid": "^8.3.2",
  "@types/jest": "^27.4.1",
  "@types/uuid": "^8.3.4",
  "jest": "^28.0.1",
  "jest-environment-jsdom": "^28.0.1",
  "typescript": "4.5.5"
}
```

Do you have any idea on what's wrong?

**Comment by broofa** [comment, marked as answer]
This is a known issue.  See #616.

**Reply by UncleVic** [reply]
I resolved it by

-- jest.config.ts
`  moduleNameMapper: {
    "^uuid$": "uuid"
  }`

-- tsconfig.spec.json
`"allowJs": true`
